modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableUtils.java

```diff
@@ -81,4 +86,68 @@ public static int findStartBuildingIndexCatalogVersion(CatalogService catalogSer
                 BUILDING, indexId, fromCatalogVersionIncluded, latestCatalogVersion
         ));
     }
+
+    /**
+     * Collects a list of tables that were removed from the catalog and should have been dropped due to a low watermark (if the catalog
+     * version in which the table was removed is less than or equal to the active catalog version of the low watermark).
+     *
+     * @param catalogService Catalog service.
+     * @param lowWatermark Low watermark, {@code null} if it has never been updated.
+     */
+    // TODO: IGNITE-21771 Process or check catalog compaction
+    static List<DroppedTableInfo> droppedTables(CatalogService catalogService, @Nullable HybridTimestamp lowWatermark) {
+        if (lowWatermark == null) {
+            return List.of();
+        }
+
+        int earliestCatalogVersion = catalogService.earliestCatalogVersion();
+        int lwmCatalogVersion = catalogService.activeCatalogVersion(lowWatermark.longValue());
+
+        var tableIds = catalogService.tables(lwmCatalogVersion).stream()
```

Comment:
This `var` usage is illegal, according to the code style

Reply:
fix it